cmd/contact-auditor/README.md

```diff
@@ -0,0 +1,75 @@
+# Contact-Auditor
+Audits subscriber registrations for e-mail addresses that
+`notify-mailer` is currently configured to skip.
+
+# Usage:
+
+```shell
+  -config string
+      File containing a JSON config.
+  -grace duration
+      Include contacts of subscribers with certificates that expired <grace>
+       period from now (default 48h0m0s)
```

Comment:
It seems like we should just run this across _all_ accounts, regardless of whether they have current certificates.

Reply:
I'll let you make the call on that.

Reply:
Let's do all accounts.